Makefile

```diff
@@ -119,7 +119,7 @@ generate:
 
 # Assumes that the `docs` checkout is a sibling of the `epinio` checkout
 generate-cli-docs:
-	@./scripts/cli-docs-generate.sh ../docs/src/references/cli
+	@./scripts/cli-docs-generate.sh ../docs/docs/references/cli
```

Comment:
Is this correct? What is the old path used for?

Reply:
The `docs/src` became `docs/docs` in the move to docasaurus. The old path is gone.

Reply:
So, IMHO this change is correct.